pkg/externalauthprovider/flags.go

```diff
@@ -256,7 +256,7 @@ func GetExternalAuthOptions(
 	if interactive.Enabled() && !cmd.Changed(claimMappingUsernameClaimFlag) {
 		result.claimMappingUsernameClaim, err = interactive.GetString(interactive.Input{
 			Question: "Claim mapping username",
-			Default:  result.claimMappingUsernameClaim,
+			Default:  "email",
```

Comment:
NP: Would it be possible to use a const here instead? 
const name like "defaultClaimMappingUsername"?